node/src/chain_spec.rs

```diff
@@ -169,6 +192,101 @@ pub fn foucoco_config() -> FoucocoChainSpec {
 				sudo_account.clone(),
 				foucoco::PARACHAIN_ID.into(),
 				false,
+				vec![],
+			)
+		},
+		// Bootnodes
+		Vec::new(),
+		// Telemetry
+		None,
+		// Protocol ID
+		Some("foucoco"),
+		// Fork ID
+		None,
+		// Properties
+		Some(properties),
+		// Extensions
+		ParachainExtensions {
+			relay_chain: "kusama".into(), // You MUST set this to the correct network!
+			para_id: foucoco::PARACHAIN_ID,
+		},
+	)
+}
+
+pub fn foucoco_standalone_config() -> FoucocoChainSpec {
+	sp_core::crypto::set_default_ss58_version(foucoco_runtime::SS58Prefix::get().into());
+
+	// Give your base currency a unit name and decimal places
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), "AMPE".into());
+	properties.insert("tokenDecimals".into(), foucoco::TOKEN_DECIMALS.into());
+	properties.insert("ss58Format".into(), foucoco_runtime::SS58Prefix::get().into());
+
+	let mut signatories: Vec<_> = foucoco::INITIAL_SUDO_SIGNATORIES
+		.iter()
+		.map(|ss58| AccountId::from_ss58check(ss58).unwrap())
+		.collect();
+	signatories.sort();
+
+	// add mock accounts to signatories so they get funded
+	// Pre-funded accounts
+	let prefunded_accounts = vec![
+		get_account_id_from_seed::<sr25519::Public>("Alice"),
+		get_account_id_from_seed::<sr25519::Public>("Bob"),
+		get_account_id_from_seed::<sr25519::Public>("Charlie"),
+		get_account_id_from_seed::<sr25519::Public>("Dave"),
+		get_account_id_from_seed::<sr25519::Public>("Eve"),
+		get_account_id_from_seed::<sr25519::Public>("Ferdie"),
+		get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
+	];
+
+	signatories.extend(prefunded_accounts.iter().cloned());
+
+	let invulnerables: Vec<_> = foucoco::INITIAL_COLLATORS
+		.iter()
+		.map(|ss58| AccountId::from_ss58check(ss58).unwrap())
+		.collect();
+
+	let sudo_account = get_account_id_from_seed::<sr25519::Public>("AltoParaíso"); //pallet_multisig::Pallet::<Runtime>::multi_account_id(&signatories[..], 3);
```

Comment:
We can of course use a different seed here. I just happened to be in the town of Alto Paraíso when I wrote this for foucoco-standalone and wasm-deploy.

Reply:
Does Alice sound good? It is easier to keep track to.

Reply:
I think it is straightforward to use Alice but I think there was a reason why I did not want to use it. I just don't remember ... let's just use Alice then.

Reply:
I changed to Alice, but we should remember also to change in `wasm-deploy` locally.